packages/front-generator/src/generators/react-typescript/entity-management/index.ts

```diff
@@ -105,11 +107,27 @@ export function answersToManagementModel(answers: EntityManagementAnswers,
     listView: answers.listView,
     editView: answers.editView,
     editAttributes: attributesInView,
-    editRelations: getRelations(projectModel, attributesInView)
+    editRelations: relations,
+    relationImports: getRelationImports(relations, entity)
   }
 }
 
-function getRelations(projectModel: ProjectModel, attributes: EntityAttribute[]): EditRelations  {
+export function getRelationImports(relations: EditRelations, entity: EntityTemplateModel): RelationImport[] {
+  const entities: EntityTemplateModel[] = Object.values(relations);
+  entities.unshift(entity);
+  return entities
+    .map(({className, path}) => {return {className, path}})
+    // todo - need to think about className collision here (same className with different path)
```

Comment:
I don't get why do we need this `.map` step. We can use `fqn` as more unique identifier

Reply:
.map step removed